src/app/shell/personal-cabinet/applications/application-card/info-status/info-status.component.scss

```diff
@@ -0,0 +1,126 @@
+@import "src/app/shared/styles/application-statuses.scss";
+
+.status-menu,
+.application-popup-status,
+.mat-application-menu,
+.application-info-menu {
+  i {
+    padding-right: 5px;
+  }
+  width: max-content;
+  max-width: max-content;
+  font-size: 13px;
+  font: Open Sans;
+  font-weight: normal;
+}
+.application-popup-status {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+  font-size: 13px;
+}
+.application-info-description-card {
+  color: #aaaaaa;
+}
+.application-info-menu,
+::ng-deep .mat-application-menu {
+  width: 60vw;
+  max-width: fit-content;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+.application-info-menu {
+  display: flex;
+  flex-direction: row;
+  max-width: max-content;
+  font-size: 13px;
+  line-height: 17px;
+}
+
+.card-block {
+  margin: 0px;
+  &-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+}
+.material-icons {
+  font-size: 16px;
+  line-height: 14.6px;
+  color: #888;
+  cursor: pointer;
+}
+
+.application-status {
+  width: 80%;
+}
+.status-icon {
+  font-size: 12px!important;
+}
+
+@media (min-width: 751px) and (max-width: 950px) {
+  .card-block {
+    max-width: 25%;
+  }
+  .card-block-buttons {
+    flex-flow: wrap;
+    row-gap: 1rem;
+  }
+  .hidden {
+    display: none !important;
+  }
+}
+
+@media(max-width: 750px) {
+  .card{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start !important;
+  }
+
+  .card>:nth-of-type(2) {
+    max-width: 100%;
+  }
+
+  .card-block,
+  .workshop-description,
+  .application-status,
+  .workshop-header,
+  .status-menu {
+    margin: 0px;
+    width: 100%;
+    text-align: center;
+    &-buttons {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-around;
+      align-items: center;
+      .btn{
+        min-width: min-content;
+        margin: 1rem 0 0 0;
```

Comment:
I belive tehre are too much redundant styles, pleas,e consider to rewrite them and add only those that are needed